extensions/arc/src/ui/dashboards/postgres/postgresParametersPage.ts

```diff
@@ -249,6 +261,42 @@ export class PostgresParametersPage extends DashboardPage {
 		]).component();
 	}
 
+	private initializeConnectButton() {
+
+		this.connectToServerButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			label: loc.connectToServer,
+			enabled: false,
+			CSSStyles: { 'max-width': '125px' }
+		}).component();
+
+		this.disposables.push(
+			this.connectToServerButton!.onDidClick(async () => {
+				this.connectToServerButton!.enabled = false;
+				if (!vscode.extensions.getExtension('microsoft.azuredatastudio-postgresql')) {
+					vscode.window.showErrorMessage(loc.missingExtension('PostgreSQL'));
```

Comment:
Oh you do already have the check here - I'd suggest updating the message to include the command though to make it easier for the user to get it installed